ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Superuser/tasks/main.yml

```diff
@@ -43,6 +43,15 @@
     - ansible_distribution == "MacOSX"
   tags: [superuser, adoptopenjdk]
 
+- name: Create Superuser account group for SLES15/s390x
+  ansible.builtin.group:
+    name: zeus
+    state: present
+  when:
+    - (ansible_distribution_major_version == "15" and ansible_architecture == "s390x")
```

Comment:
Is there a reason this is listed as specific to s390x and 15? Would distribution SLES or similar not be ok?

Reply:
I'll make this generic for s390x, I was just being a little too precise, and ensuring my changes only reflected what I'd tested. I'll make this generic, and run a test on the sles12 box.